### ISSUE TITLE
Add KML export Sandcastle example

### ISSUE THREAD

**Issue by OmarShehata** [AnalyticalGraphicsInc/cesium#7964]
The PR https://github.com/AnalyticalGraphicsInc/cesium/pull/7921 adds a function to export an EntityCollection in Cesium as KML/KMZ document. There's an example in the doc, but it would be nice to have a full code example that includes the required blob downloading as well. See Fili's code snippet here: https://github.com/AnalyticalGraphicsInc/cesium/pull/7921#issuecomment-505019659

It would be cool to use one of these CZML's as a source: https://cesiumjs.org/Cesium/Apps/Sandcastle/index.html?src=CZML.html

Or the extruded GeoJSONs here, so the example shows you how to import a GeoJSON, modify it, then export it:

https://cesiumjs.org/Cesium/Apps/Sandcastle/index.html?src=GeoJSON%20and%20TopoJSON.html

Would also be nice to show how to use the `modelCallback` to handle rewriting the model URLs. This would be a great first contribution I think!

**Comment by pjcozzi** [comment]
@tfili @OmarShehata why didn't #7921 include a Sandcastle example?  We usually would not merge a new feature without one.

**Comment by tfili** [comment]
I thought it was a little weird to have a Sandcastle example that just downloads a file that you then need another piece of desktop software to open. I opted for the example in the doc and others seemed to agree. I didn't realize this was going to be a TODO or I would have spent the 10 minutes doing it since the code was already written.

**Comment by pjcozzi** [comment]
The KML can be dragged and dropped back into Cesium, right?  What other software does it need?  To make the feature discoverable seems like we should have went with a Sandcastle example, right?  I get that there are limitations, but that is fine if they are documented.

**Comment by OmarShehata** [comment]
I did think a Sandcastle example would be useful, especially to show the download code + rewriting model URLs which may not be obvious. I merged without waiting on that because I figured we could add a fuller example than the one in the doc later, so that was my bad.

**Comment by tfili** [comment]
Add one here: https://github.com/AnalyticalGraphicsInc/cesium/pull/7970
